llvm/test/CodeGen/RISCV/bfloat-imm.ll

```diff
@@ -1,14 +1,15 @@
 ; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py UTC_ARGS: --version 2
 ; RUN: llc -mtriple=riscv32 -mattr=+experimental-zfbfmin -verify-machineinstrs \
-; RUN:   -target-abi ilp32f < %s | FileCheck %s
+; RUN:   -target-abi ilp32f < %s | FileCheck --check-prefixes=CHECK %s
```

Comment:
Why do we need to add --check-prefixes here?

Reply:
We don't.  I think this was a bad rebase.  New version pending shortly.